src/js/components/AppTaskStatsListComponent.jsx

```diff
@@ -39,7 +39,7 @@ var AppTaskStatsListComponent = React.createClass({
     var taskStatsList = this.props.taskStatsList;
     var noTaskStatistics = null;
 
-    if (taskStatsList == null) {
+    if (taskStatsList == null || !Object.keys(taskStatsList).length) {
```

Comment:
Tiny nit, but can we use `=== 0` instead of `!length`? Just to make the intention explicit.